lib/babe/build_test.go

```diff
@@ -236,6 +237,8 @@ func TestBuildBlock_failing(t *testing.T) {
 	if err == nil {
 		t.Fatal("should error when attempting to include invalid tx")
 	}
+	require.Equal(t, "cannot build extrinsics: Error during apply extrinsic: Apply error, type: Payment",
+		err.Error(), "Did not recieve expected error text")
```

Comment:
Should I try to create more tests for failing apply extrinsic to test for different failure types?

Reply:
it wouldn't hurt to throw a couple more in :+1: